assessment_helpers/plot_csv.py

```diff
@@ -0,0 +1,14 @@
+#This script uses seaborn to plot "clean_ga_data.csv" (aka the output from to_csv.py)
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pandas as pd
+
+sns.set_theme(color_codes=True)
+
+plotData = pd.read_csv("clean_GA_Data.csv")
+
+#ax = sns.regplot(x="Generation", y="Best Fitness", data=plotData)
+#ax = sns.regplot(x="Generation", y="Average Fitness", data=plotData)
+ax = sns.regplot(x="Generation", y="Average Score", data=plotData)
+plt.show()
```

Comment:
I think that you should wrap this in order to make it easier to run from the command line. Here is one way of doing it:
```suggestion
import argparse

if __name__ == '__main__':
    parser = argparse.ArgumentParser()
    parser.add_argument("--csv_filename", nargs='?', type=str, default="clean_GA_Data.csv")
    parser.add_argument("--figure_filename", nargs='?', type=str, default="")
    args = parser.parse_args()
    
    sns.set_theme(color_codes=True)
    
    plotData = pd.read_csv(args.csv_filename)

    #ax = sns.regplot(x="Generation", y="Best Fitness", data=plotData)
    #ax = sns.regplot(x="Generation", y="Average Fitness", data=plotData)
    ax = sns.regplot(x="Generation", y="Average Score", data=plotData)
    plt.show()
    
    if (args.figure_filename):
        plt.savefig(args.figure_filename)
```
with this change, from the command line you should be able to type `python plot_csv.py --csv_filename="clean_GA_Data.csv"`. This makes it more flexible since we aren't hardcoding filenames (although we do still have them as defaults.

I also added plt.savefig. This means that if somebody does `python plot_csv.py --csv_filename="file.csv" --figure_filename="plot.png" the function will save an actual image that we can view.